cmd/main.go

```diff
@@ -84,8 +84,8 @@ func main() {
 		"If set the metrics endpoint is served securely")
 	flag.BoolVar(&enableHTTP2, "enable-http2", false,
 		"If set, HTTP/2 will be enabled for the metrics and webhook servers")
-	flag.BoolVar(&featureGatePrometheusOperator, "feature-gate.enable-service-monitors", true,
-		"If set, the operator will manage ServiceMonitors for Prometheus Operator")
+	flag.BoolVar(&featureGatePrometheusOperator, "feature-gate.enable-prometheus-operator-crds", true,
+		"If set, the operator will manage ServiceMonitors for components it deploys, and discover PrometheusRule objects to set on Thanos Ruler, from Prometheus Operator.")
```

Comment:
this is good for now, but probably want to move the more detailed description of behaviour into docs at some point and we can keep this brief!

Reply:
Added a section!